service/gcs/prot/protocol.go

```diff
@@ -306,14 +306,6 @@ func UnmarshalContainerModifySettings(b []byte) (*ContainerModifySettings, error
 		return nil, errors.WithStack(err)
 	}
 
-	// Fill in default fields.
-	if request.Request.ResourceType == "" {
-		request.Request.ResourceType = PtMemory
-	}
-	if request.Request.RequestType == "" {
-		request.Request.RequestType = RtAdd
-	}
-
```

Comment:
For `RequestType` I guess you are correct. We should keep the removal of `ResourceType` though because its not `[OmitEmpty]`